email_account_validity/_base.py

```diff
@@ -18,13 +18,14 @@
 
 from twisted.web.server import Request
 
-from synapse.api.errors import StoreError, SynapseError
 from synapse.http.servlet import parse_json_object_from_request
 from synapse.module_api import ModuleApi
+from synapse.module_api.errors import SynapseError
 from synapse.types import UserID
 from synapse.util import stringutils
```

Comment:
there are a bunch of imports here of things that aren't in the module api. I think most of them are small enough to be re-implemented; those that aren't need adding to `module_api`.